imports/startup/client/clientError.js

```diff
@@ -0,0 +1,50 @@
+const reportToServer = function(error) {
+	var report =
+		{ name: error.name
+		, message: error.message
+		, location: window.location.href
+		, tsClient: new Date()
+		};
+	Meteor.call('clientError', report);
+}
+
+window.addEventListener("error", function(event) {
+	reportToServer(event.error);
+});
+
+var buffer = [];
+var discriminatoryReporting = function(arguments) {
+	var msg = arguments[0];
+
+	// "Exception from Tracker recompute function:"
+	if (msg.indexOf("Exception ") === 0) {
+		// Boring
+		return;
+	}
```

Comment:
Why is that boring?

Reply:
@sbalmer Can't you make it a bit more enthralling? Almost fell asleep...

Reply:
Yea could have left out those lines. I was actually almost asleep when I wrote this :-) So I had that dream of writing a handler for all possible strings that get thrown at `_debug()`. In that context I called the `"Exception from Tracker"` line boring because it wasn't adding information to the following `"Error: No such function: ..."` line.

Reply:
Oups that should be msg.indexOf("Exception from Tracker") or something